packages/office-ui-fabric-react/src/components/KeytipLayer/KeytipLayer.styles.ts

```diff
@@ -0,0 +1,22 @@
+import { ILayerStyles, ILayerStyleProps } from '../../Layer';
+import { IKeytipLayerStyleProps, IKeytipLayerStyles } from './KeytipLayer.types';
+import { hiddenContentStyle, ZIndexes } from '../../Styling';
+
+export const getLayerStyles = (props: ILayerStyleProps): ILayerStyles => {
+  return {
+    root: [{
+      // Prioritize the Keytips above all other Layers
+      zIndex: ZIndexes.KeytipLayer
```

Comment:
do not use z-index, rely on callout for proper layering

Reply:
The reason I can't use the regular Layer ordering is because all of our Keytips (Callouts) live in one Layer (our KeytipLayer). The KeytipLayer is created at the beginning of the app and so every other Layer created after will be appended to the document body, which will make any ContextualMenu, etc, overlap the Keytips. We always want Keytips to be above all components, so this was the only solution I found. I talked to David about adding a "priority" prop to Layer to determine the ordering in the future, but it would require more work than I have time for right now